pythonClient/testProfile.py

```diff
@@ -0,0 +1,68 @@
+import requests
+from requests.api import head
+from requests.auth import HTTPBasicAuth
+import json
+
+staticHeader = \
+"""
+<!DOCTYPE HTML>
```

Comment:
Is this really how flask has you deal with HTML?

Reply:
Nope